### PR TITLE
[JBPM-10084]-Remove references of the start.jbpm.org in docs

### PR DESCRIPTION
**Pull request by VaniHaripriya**
Jira- https://issues.redhat.com/browse/JBPM-10084
Removed all the references for start.jbpm.org in Gettingstarted doc

### CODE REVIEW

File: content/businessapps/gettingStarted.adoc [hunk 1]
```diff
@@ -39,20 +39,68 @@ data model classes something more than they are - include too much of implementa
...
+
+
+== Build your first Business Application
 
-There are two options you can get your application
+To build your first business application,use the above mentioned maven archetype commands to generate a Data project,Business assets project and a Service project with desired capabilities.
 
-* Generate default application - that gives you all default settings to get started as fast as possible
-* Configure projects and capabilities - that gives you few options to select from before generating application
+It is recommended to generate all the three types of projects -Data Model, Business assets and Service projects using the archetypes in same directory on your computer.
```
Comment (gmunozfe) [inline content/businessapps/gettingStarted.adoc:100]: Missing closing dash

File: content/businessapps/gettingStarted.adoc [hunk 2]
```diff
@@ -39,20 +39,68 @@ data model classes something more than they are - include too much of implementa
...
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-service-spring-boot-archetype -DarchetypeVersion={COMMUNITY_VERSION_FINAL} -DgroupId=com.company -DartifactId=business-application-service -Dversion=1.0-SNAPSHOT -Dpackage=com.company.service -DappType=planner
+----
+`org.kie:kie-service-spring-boot-archetype:{COMMUNITY_VERSION_FINAL}`
+
+
+== Build your first Business Application
 
-There are two options you can get your application
+To build your first business application,use the above mentioned maven archetype commands to generate a Data project,Business assets project and a Service project with desired capabilities.
```
Comment (gmunozfe) [inline content/businessapps/gettingStarted.adoc:98]: Missing blanks after comma

**Review by gmunozfe — APPROVED** [review]
Looks good to me, just minor format comments

File: content/businessapps/gettingStarted.adoc [hunk 3]
```diff
@@ -39,20 +39,68 @@ data model classes something more than they are - include too much of implementa
...
+
+`org.apache.maven.archetypes:maven-archetype-quickstart:1.3`
+
+* Business assets project archetype
+
+[source, bash,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-kjar-archetype -DarchetypeVersion={COMMUNITY_VERSION_FINAL} -DgroupId=com.company -DartifactId=business-application-kjar -Dversion=1.0-SNAPSHOT -Dpackage=com.company
+----
+`org.kie:kie-kjar-archetype:{COMMUNITY_VERSION_FINAL}`
```
Comment (hbelmiro) [inline content/businessapps/gettingStarted.adoc:59]: This doesn't work as you expect. The variables aren't resolved when you build the website.

Due to a bug in JBake, the only way I managed to make variables work is by converting the adoc content to ftl. You can see how I did it in https://github.com/kiegroup/jbpm-website/blob/main/templates/_content_businessapps_examples.ftl. This template is used by https://github.com/kiegroup/jbpm-website/blob/main/content/businessapps/examples.adoc

You can use https://asciidoclive.com/edit/scratch/1 to convert the adoc code to HTML. You can see the generated HTML by clicking on the gear icon in the top right corner and then click in "Show output HTML".

**Review by hbelmiro — CHANGES_REQUESTED** [review]
@VaniHaripriya the variables are not being resolved. I left a comment in the code with the explanation. Let me know if you need any help with that.

File: content/businessapps/gettingStarted.adoc (hunk 3 above)
Comment (VaniHaripriya) [reply content/businessapps/gettingStarted.adoc]: Thank you Helber, made the changes as suggested.

**Review by hbelmiro — APPROVED** [review]
Great work @VaniHaripriya.
LGTM. Thank you.

File: templates/_content_businessapps_gettingStarted.ftl [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                in the project. Capabilities essentially define the features that your business
+                application will be equipped with. Available options are:</p>
+            </div>
+            <div class="ulist">
+              <ul>
+                <li>
+                  <p>Business automation(bpm) covers features for process management, case management,
+                    decision management and optimization. These will be by default configured
+                    in the service project of your business application. Although you can
+                    turn them off via configurationService project archetype(Business Automation)</p>
```
[the remark this answers is not in the document]
Comment (VaniHaripriya) [reply templates/_content_businessapps_gettingStarted.ftl:156]: That line was not supposed to be there Alessandro. Removed it now.

**Review by lazarotti — APPROVED** [review]

**Comment by lazarotti** [comment]
all good for me @VaniHaripriya 
Let me know when the site is updated, so I can ask to redirect the start.jbpm.org to this page

**Comment by VaniHaripriya** [comment]
> all good for me @VaniHaripriya Let me know when the site is updated, so I can ask to redirect the start.jbpm.org to this page

Thank you Alessandro, I will let you know once it is updated.

**Review by mbiarnes — APPROVED** [review]

**Pull request merged**